app/controllers/members_controller.rb

```diff
@@ -300,4 +271,12 @@ def sort_by_groups_last(members)
 
     members.sort_by { |m| group_ids.include?(m.user_id) ? 1 : -1 }
   end
+
+  def display_error(service_call)
```

Comment:
We could also extract these into a `ServicedController` (or similar) concern that then gets included. We have this pattern quite a number of times already with the "respond with success"

Reply:
Definitely, @oliverguenther. I have been meaning to do that for a long time so if you have the breath, go ahead. Ideally, we would have something similar to the API endpoints where you have an enforced standard on what a controller does.